recipes/agent.rb

```diff
@@ -35,6 +37,11 @@
   # aptitude is required to get available update count
   package 'aptitude'
 
+  directory '/etc/zabbix/zabbix_agentd.d' do
```

Comment:
On my machines, this directory is maintained by the apt packages. I suppose it doesn't hurt to re-define it here though. Wasn't it created in your tests?